LibraryManagement.SqlServerDatabase/DBO/Tables/Genres.sql

```diff
@@ -2,4 +2,5 @@
 (
     GenreID INT CONSTRAINT PK_Genres PRIMARY KEY IDENTITY(1,1),
     Name VARCHAR(50) NOT NULL,
+    CONSTRAINT Name_NotEmpty CHECK (LEN(Name) > 0)
```

Comment:
Lets change the name here, I looked up the naming convention for CHECK Constraints, I think we'd want something like: CHK_Genres_Name

https://www.scholarhat.com/tutorial/sqlserver/sql-server-naming-conventions-and-standards